localstack/services/events/events_listener.py

```diff
@@ -0,0 +1,67 @@
+import json
+import re
+import tempfile
+import uuid
+
+from requests.models import Response
+
+from localstack.constants import TEST_AWS_ACCOUNT_ID, MOTO_ACCOUNT_ID
+from localstack.services.generic_proxy import ProxyListener
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import to_str, save_file, TMP_FILES, mkdir
+
+EVENTS_TMP_DIR = tempfile.gettempdir() + '/cw_events/'
+
+
+class ProxyListenerEvents(ProxyListener):
+
+    def forward_request(self, method, path, data, headers):
+        if method == 'POST' and path == '/' and headers['X-Amz-Target'] == 'AWSEvents.PutEvents':
+            parsed_data = json.loads(to_str(data))
+            events_with_added_uuid = list(
+                map(lambda event: {'event': event, 'uuid': str(uuid.uuid4())}, parsed_data['Entries']))
+            response_string = json.dumps(
+                {'Entries': list(map(lambda event: {'EventId': event['uuid']}, events_with_added_uuid))})
+            self._create_and_register_temp_dir()
+            self._dump_events_to_files(events_with_added_uuid)
+            response = Response()
+            response.status_code = 200
+            response._content = response_string
+            return response
+        return True
+
+    def return_response(self, method, path, data, headers, response, request_handler=None):
+        if response.content:
+            # fix hardcoded account ID in ARNs returned from this API
+            self._fix_account_id(response)
+            # fix dates returned from this API (fixes an issue with Terraform)
+            self._fix_date_format(response)
+            # fix content-length header
+            response.headers['content-length'] = len(response._content)
+
+    def _create_and_register_temp_dir(self):
+        if EVENTS_TMP_DIR not in TMP_FILES:
+            mkdir(EVENTS_TMP_DIR)
+            TMP_FILES.append(EVENTS_TMP_DIR)
+
+    def _dump_events_to_files(self, events_with_added_uuid):
+        for event in events_with_added_uuid:
+            save_file(EVENTS_TMP_DIR + event['uuid'], json.dumps(event['event']))
```

Comment:
Better:
```
save_file(os.path.join(EVENTS_TMP_DIR, event['uuid']), json.dumps(event['event']))
```

Reply:
Thanks, I should use those os tools more. :-)